web_responsive/README.rst

```diff
@@ -0,0 +1,228 @@
+
+* Increase the size of the labels in extra large screens
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/label_size_small.png
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/label_size_large.png
+
+**Features for mobile**:
+
+* App-specific submenus are shown on full screen when toggling them from the
+  "hamburger" menu
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/hamburger.gif
+
+* User-specific submenus are shown on full screen when toggling them from the
+  "avatar" menu
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/usermenu.gif
+
+* View type picker dropdown displays comfortably
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/viewtype.gif
+
+* Top app bar is always visible, but the control panel is hidden when
+  scrolling down, to save some valuable vertical space
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/navbar.gif
+
+* Form status bar action and status buttons are collapsed in dropdowns.
+  Other control panel buttons use icons to save space.
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/form_buttons.gif
+
+* Breadcrumbs navigation is collapsed with a "back arrow" button.
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/breadcrumbs.gif
+
+* Search panel is collapsed to mobile version on small screens.
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/search_panel.gif
+
+* Followers and send button is displayed on mobile. Avatar is hidden.
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/chatter.gif
+
+* Scrollable dropdowns
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/dropdown_scroll.gif
+
+* Kanban interface adopted to mobile
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/kanban.gif
+
+* Calendar interface adopted to mobile
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/calendar.gif
+
+* Interface is adapted dynamically on device rotation
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/device_rotation.gif
+
+* Big inputs on form in edit mode
+
+**Features for desktop computers**:
+
+* Keyboard shortcuts for easier navigation,
+  **using `Alt + Shift + [NUM]`** combination instead of
+  just `Alt + [NUM]` to avoid conflict with Firefox Tab switching.
+  Standard Odoo keyboard hotkeys changed to be more intuitive or
+  accessible by fingers of one hand.
+  F.x. `Alt + S` for `Save`
+
+  .. image:: https://raw.githubusercontent.com/OCA/web/16.0/web_responsive/static/img/shortcuts.gif
+
+* Autofocus on search menu box when opening the app menu
```

Comment:
@youring , this is now resolved

Reply:
@bvanjeelharia Thank you for the quick fix. 
For the record, tested on runboat: while the tab/shift+tab key navigation works in web_responsive v14, but missing in v15...and this pr too since v16 is migrated from v15.

Reply:
@youring , the tab/shift+tab key navigation on app menu works now.

Reply:
The tab/shift tab is working, but maybe css style for active item needs fixing.
![border for active item v16 vs v14](https://user-images.githubusercontent.com/4132418/218296425-a6cdddb5-1f38-47a1-9d9a-ea7077aa4e93.png)

Another problem, if you are on the search result list, press tab, then press arrow key(up or down) which works in previous version, it will throw error
![tab-arrow error](https://user-images.githubusercontent.com/4132418/218296551-b8559d71-d02d-45a0-8dfd-cfe704c56c29.png)

Reply:
Yes, thank you. Arrow keys works on app menu search result list. But on app menu its self, still same error. Sorrry that I miss this test on the previous commit, step to reproduce: Alt+A to call the app menu, Tab, Tab, Up/Down.
```
UncaughtClientError > TypeError

Uncaught Javascript Error > can't access property "writeText", browser.navigator.clipboard is undefined

TypeError: can't access property "writeText", browser.navigator.clipboard is undefined
    onClickClipboard@http://oca-web-16-0-pr2405-675e2afaee8e.runboat.odoo-community.org/web/assets/533-76a3522/web.assets_backend.min.js:618:20
    mainEventHandler@http://oca-web-16-0-pr2405-675e2afaee8e.runboat.odoo-community.org/web/assets/509-b7eb35d/web.assets_common.min.js:2041:77
    listener@http://oca-web-16-0-pr2405-675e2afaee8e.runboat.odoo-community.org/web/assets/509-b7eb35d/web.assets_common.min.js:1274:15